addon/components/code-editor.ts

```diff
@@ -36,6 +38,17 @@ export default class CodeEditor extends Component {
     }
   }
 
+  public onEditorReady() {
+    const iframe: any = this.element.querySelector<HTMLIFrameElement>(
+      '.frame-container > iframe'
+    )
+    const editor = (<any>iframe.contentWindow).editor
+
+    if (this.onReady) {
+      this.onReady(editor)
+    }
```

Comment:
Not a big deal with this one change, but if we end up adding more hooks like this, I'd quickly start to fail initializing the optional functions with no-ops

```ts
  public onReady?: (editor: mon.editor.IStandaloneCodeEditor) => any = () => null;
```

so that we can just invoke them directly without worrying too much about their presence/absence